core/lib/common/suite.js

```diff
@@ -244,12 +243,7 @@ class Suite {
 			// Teardown all running test suites before removing assets & dependencies
 			this.state.log(`Test suite completed. Tearing down now.`);
 			await this.teardown.runAll();
-			await this.createJsonSummary();
 			await this.removeDependencies();
-			// This env variable can be used to keep a configured, unpacked image for use when developing tests
-			if (process.env.DEBUG_KEEP_IMG !== true) {
-				await this.removeDownloads();
-			}
```

Comment:
Can you add this back in, I am removing it some other PR and it will create a conflict for no reason.